jupyterhub/alembic/versions/56cc5a70207e_token_tracking.py

```diff
@@ -0,0 +1,42 @@
+"""token tracking
+
+Revision ID: 56cc5a70207e
+Revises: 1cebaf56856c
+Create Date: 2017-12-19 15:21:09.300513
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '56cc5a70207e'
+down_revision = '1cebaf56856c'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    tables = op.get_bind().engine.table_names()
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Under what circumstances would this be adjusted? Different data fields stored for tokens?

Reply:
This is an auto-generated comment from `alembic revision --autogenerate`. I should have removed it after making sure it was correct.